source/mqtt-tls.rst

```diff
@@ -0,0 +1,341 @@
+    (by running commands like ``ps aux | grep mosquitto``) to avoid port conflicts.
+
+On Debian-based Linux distributions you can install
+``mosquitto-clients``  running:
+
+.. code:: bash
+
+   sudo apt-get install mosquitto-clients
+
+
+Broker MQTT IotAgent VerneMQ (Default)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To publish and subscribe using the appropriate certificates,
+you must have IotAgent VerneMQ Broker, V2K Bridge,
+K2V Bridge and the X.509 Identity Management running,
+see more in `IotAgent-VerneMQ`_.
+
+Simulating a device publishing with mosquitto
+
+.. code:: bash
+
+   mosquitto_pub  -h localhost -p 8883 -t <tenant>:<deviceId>/attrs  -m '{"attr_example": 10}' --cert <device .crt file> --key <device .key file> --cafile <ca .crt file>
+
+An example of publication with the certificates and
+keys generated in the previous topic with tool `CertReq`_.
+
+.. code:: bash
+
+   mosquitto_pub \
+   -h localhost \
+   -p 8883 \
+   -t admin:a1998e/attrs \
+   -m '{"attr_example": 10 }' \
+   --cert './cert_a1998e/cert.pem' \
+   --key './cert_a1998e/private.key' \
+   --cafile './ca/ca.pem'
+
+Simulating a device subscribing with mosquitto
+
+.. code:: bash
+
+   mosquitto_sub  -h localhost -p 8883 -t <tenant>:<deviceId>/config  --cert <device .crt file> --key <device .key file> --cafile <ca .crt file>
+
+For more details about simulate a device see
+in `Simulating a device with mosquitto`_
+and more about simulate a device with security
+in `Simulating a device with mosquitto with security`_.
+
+Broker MQTT IotAgent Mosca (Legacy)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To publish and subscribe using the appropriated certificates,
+you must need to be with the IotAgent Mosca Broker and
+the X.509 Identity Management running,
+see more in `IotAgent-Mosca`_.
+In addition, you need to use a **different topic** from VerneMQ
+and pass the identifier to publish and subscribe, as follows:
+
+How to publish:
+
+.. code:: bash
+
+   mosquitto_pub  -h localhost -p 8883 -t /<tenant>/<deviceId>/attrs -i <tenant>:<deviceId> -m '{"attr_example": 10}' --cert <device .crt file> --key <device .key file> --cafile <ca .crt file>
+
+How to subscribe:
+
+.. code:: bash
+
+   mosquitto_sub  -h localhost -p 8883 -t /<tenant>/<deviceId>/config -i <tenant>:<deviceId> --cert <device .crt file> --key <device .key file> --cafile <ca .crt file>
+
+
+Note: In all those examples above, the message is a publication
+with an attribute, this attribute has the label `attr_example`
+and a new value 10, you need to change this for your case.
+
+Unsecured mode MQTT (without TLS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**MQTT without security is not recommended, use this for testing only.**
+
+In *ansible-dojot* (kubernetes deployment)  you can disable the `unsecured mode`
+by changing the ``dojot_insecure_mqtt`` variable to ``false``,
+this is valid in both Brokers.
+Check the :doc:`./installation-guide` for more info.
+
+
+Broker MQTT IotAgent VerneMQ (Default)
```

Comment:
Sounds a bit strange to me, maybe only `VerneMQ (default)` as a title would do the job.